config/crd/bases/atlas.mongodb.com_atlasprojects.yaml

```diff
@@ -311,6 +311,38 @@ spec:
                       type: string
                   type: object
                 type: array
+              projectMaintenanceWindow:
```

Comment:
I think we can rename that to be just `maintenanceWindow`, because it is already a part of a `project` resource

Reply:
Should we do the same with `projectIpAccessList` for consistency then ?